### PR TITLE
Change the default image in manifests to target release-0.4

### LINKED ISSUE #1205: [0.4] Change the default image in manifests to target release-0.4
Now that master is open to v1alpha3, we want to change the tag referenced by default https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/release-0.4/config/default/manager_image_patch.yaml#L11 to be `release-0.4`, given that `latest` would yield the wrong image.

/kind feature
/priority important-soon
/milestone v0.5.0
/help

### PR DESCRIPTION
**Pull request by tahsinrahman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1205

### CODE REVIEW

File: config/default/manager_image_patch.yaml [hunk 1]
```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller:latest
+      - image: gcr.io/k8s-staging-cluster-api-aws/cluster-api-aws-controller:release-0.4
```
Comment (detiber) [inline config/default/manager_image_patch.yaml]: We're not yet building images on postsubmit for the `release-0.4` branch yet.. @vincepri

Comment (vincepri) [reply config/default/manager_image_patch.yaml]: We'll have to enable it in `test-infra` first, if you wanna do that @tahsinrahman

Comment (tahsinrahman) [reply config/default/manager_image_patch.yaml]: I don’t have any experience  on test-infra, can you give me some references? Thanks!

Comment (ncdc) [reply config/default/manager_image_patch.yaml]: https://github.com/kubernetes/test-infra/pull/14482/files

**Comment by ncdc** [comment]
/hold
pending test-infra changes to build the release-0.4 branch

File: config/default/manager_image_patch.yaml (hunk 1 above)
Comment (tahsinrahman) [reply config/default/manager_image_patch.yaml]: Thanks!
Is it ok? https://github.com/kubernetes/test-infra/pull/14744

**Comment by tahsinrahman** [comment]
https://github.com/kubernetes/test-infra/pull/14744 is merged!

**Pull request merged**